charts/sonarqube/templates/sonarqube-sts.yaml

```diff
@@ -111,6 +111,10 @@ spec:
             - mountPath: /data
               name: sonarqube
               subPath: data
+          {{- with .Values.env }}
```

Comment:
any reason you need to have the general env here and not something specific to the scope of the init container? i think it would make more sense to create a separate scope for this if this is even necessary

Reply:
Agree. Something more specific would be better. Will update

When running in environments where the only path to internet is through proxy, some way to specify the proxy settings is needed to get the prometheus init container to complete.

Reply:
Realized that the  mountpoint  /opt/sonarqube/conf was actually used twice when activating prometheus export. No go :-)
Moved prometheus.yaml to sonarqube config instead.

Also, busybox wget does not support https proxy. Using curl instead.